bin/webpack.js

```diff
@@ -1,19 +1,43 @@
 #!/usr/bin/env node
 
+const { exec } = require("child_process");
+const readline = require("readline");
+
+let io = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
 let webpackCliInstalled = false;
+
 try {
-	require.resolve("webpack-cli");
+	require.resolve("webpack-cli") &&
+	require("webpack-cli");
```

Comment:
Don't require webpack-cli in the try block. This catches errors on loading too. We don't want this.

Do it outside of the try block.